crates/biome_graphql_parser/tests/graphql_test_suite/err/definitions/input_object_extension.graphql.snap

```diff
@@ -0,0 +1,151 @@
+                        ty: GraphqlNamedType {
+                            name: GraphqlName {
+                                value_token: GRAPHQL_NAME@59..64 "Float" [] [],
+                            },
+                        },
+                        default: missing (optional),
+                        directives: GraphqlDirectiveList [],
+                    },
+                ],
+                r_curly_token: R_CURLY@64..66 "}" [Newline("\n")] [],
+            },
+        },
+    ],
+    eof_token: EOF@66..67 "" [Newline("\n")] [],
+}
+```
+
+## CST
+
+```
+0: GRAPHQL_ROOT@0..67
+  0: (empty)
+  1: GRAPHQL_DEFINITION_LIST@0..66
+    0: GRAPHQL_INPUT_OBJECT_TYPE_EXTENSION@0..20
+      0: EXTEND_KW@0..7 "extend" [] [Whitespace(" ")]
+      1: INPUT_KW@7..13 "input" [] [Whitespace(" ")]
+      2: GRAPHQL_NAME@13..20
+        0: GRAPHQL_NAME@13..20 "Point2D" [] []
+      3: GRAPHQL_DIRECTIVE_LIST@20..20
+      4: (empty)
+    1: GRAPHQL_INPUT_OBJECT_TYPE_EXTENSION@20..66
+      0: EXTEND_KW@20..29 "extend" [Newline("\n"), Newline("\n")] [Whitespace(" ")]
+      1: INPUT_KW@29..35 "input" [] [Whitespace(" ")]
+      2: GRAPHQL_NAME@35..42
+        0: GRAPHQL_NAME@35..42 "Point2D" [] []
+      3: GRAPHQL_DIRECTIVE_LIST@42..42
+      4: GRAPHQL_INPUT_FIELDS_DEFINITION@42..66
+        0: (empty)
+        1: GRAPHQL_INPUT_FIELD_LIST@42..64
+          0: GRAPHQL_INPUT_VALUE_DEFINITION@42..53
+            0: (empty)
+            1: GRAPHQL_NAME@42..46
+              0: GRAPHQL_NAME@42..46 "x" [Newline("\n"), Whitespace("  ")] []
+            2: COLON@46..48 ":" [] [Whitespace(" ")]
+            3: GRAPHQL_NAMED_TYPE@48..53
+              0: GRAPHQL_NAME@48..53
+                0: GRAPHQL_NAME@48..53 "Float" [] []
+            4: (empty)
+            5: GRAPHQL_DIRECTIVE_LIST@53..53
+          1: GRAPHQL_INPUT_VALUE_DEFINITION@53..64
+            0: (empty)
+            1: GRAPHQL_NAME@53..57
+              0: GRAPHQL_NAME@53..57 "y" [Newline("\n"), Whitespace("  ")] []
+            2: COLON@57..59 ":" [] [Whitespace(" ")]
+            3: GRAPHQL_NAMED_TYPE@59..64
+              0: GRAPHQL_NAME@59..64
+                0: GRAPHQL_NAME@59..64 "Float" [] []
+            4: (empty)
+            5: GRAPHQL_DIRECTIVE_LIST@64..64
+        2: R_CURLY@64..66 "}" [Newline("\n")] []
+  2: EOF@66..67 "" [Newline("\n")] []
+
+```
+
+## Diagnostics
+
+```
+input_object_extension.graphql:3:1 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Expected at least one directive or a set of fields definition
+  
+    1 │ extend input Point2D
+    2 │ 
+  > 3 │ extend input Point2D
+      │ ^^^^^^
+    4 │   x: Float
+    5 │   y: Float
+  
+input_object_extension.graphql:4:3 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × expected `{` but instead found `x`
+  
+    3 │ extend input Point2D
+  > 4 │   x: Float
+      │   ^
+    5 │   y: Float
+    6 │ }
+  
+  i Remove x
```

Comment:
I feel this suggestion could be improved.

Reply:
I will create a new PR to address this, as the current PR does not modify this behavior but only copies the error message from the input object definition rule. What would you suggest for the new suggestion?

Reply:
It seems the code could be fixed by adding the `{`, so maybe a suggestion along the lines of “did you forget to add the `{`”?